core/templates/dev/head/services/FilterServiceSpec.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the Alerts Service.
+ */
+
+describe('Filter Service', function() {
+  var FilterService = null;
+
+  beforeEach(module('oppia'));
+  beforeEach(inject(function($injector) {
+    FilterService = $injector.get(
+      'FilterService');
+  }));
+
+  it('should not shorten the length of text if less than 50', function() {
+    expect(FilterService.getAbbreviatedText('Lorem ipsum dolor'))
```

Comment:
While it is fine to use lorem ipsum in manual tests, we try to avoid those in automated tests, so you'll want to use an actual sentence.

Reply:
Done